Testing/Data/SystemTest/ISIS_Powder/input/HRPD66063_focused.nxs.md5

```diff
@@ -1 +0,0 @@
-108ad48ccf2804be44f860741c380e8c
```

Comment:
It's good we deleted this to avoid confusion with the duplicate in the reference directory.